server/README.md

```diff
@@ -0,0 +1,211 @@
+---
+
+copyright:
```

Comment:
Like mentioned, its copyright from IBM; just leave it there for acknowledgment since we used their starter kit. I am so sure about how all of this copyright/freeuse policies things will have effects on us.

![image](https://user-images.githubusercontent.com/18486562/88123169-2fb68f80-cb7f-11ea-9fcb-9eaedfd224e7.png)